### PR TITLE
Cleanup includes (part II)

### PR DESCRIPTION
**Pull request by ace20022**

### CODE REVIEW

**Comment by ace20022** [comment]
jenkins build this please

**Comment by Paxxi** [comment]
Have you been using clion or resharper? :)

File: xbmc/Application.cpp [hunk 1]
```diff
@@ -141,18 +129,14 @@
 #endif
 #ifdef HAS_JSONRPC
 #include "interfaces/json-rpc/JSONRPC.h"
-#include "network/TCPServer.h"
 #endif
 #ifdef HAS_AIRPLAY
-#include "network/AirPlayServer.h"
 #endif
```
Comment (Paxxi) [inline xbmc/Application.cpp]: This Ifdef should be removed as well

File: xbmc/Application.cpp [hunk 2]
```diff
@@ -141,18 +129,14 @@
...
 #ifdef HAS_AIRTUNES
```
Comment (Paxxi) [inline xbmc/Application.cpp]: Same here

**Comment by Paxxi** [comment]
+1 btw

**Comment by ace20022** [comment]
> Have you been using clion or resharper? :)

ReSharper trial ;)

**Comment by ace20022** [comment]
any objections?

jenkins build this please

**Comment by ace20022** [comment]
Some addons failed, not sure why the red cross appeared...

**Comment by MartijnKaijser** [comment]
jenkins build this please

**Comment by ace20022** [comment]
Since I can't guarantee that behaviour doesn't change this is for I++.

**Comment by Paxxi** [comment]
I fail to see how this would change behaviour but since it's only useful for us devs it seems reasonable to be cautious

**Comment by ace20022** [comment]
I have not checked (re-)defines, besides, e.g., system.h. So I just don't want to be responsible for issues in that stage ;)

**Comment by ace20022** [comment]
All files that are visible in the vs project checked

jenkins build this please

**Comment by MartijnKaijser** [comment]
damn, that's some cleanup :)

**Comment by MartijnKaijser** [comment]
@ace20022 i'm ok with this one going in as we are waiting for other fixes. Any problem should already have shown on building. 
Care to rebase?

**Comment by ace20022** [comment]
jenkins build this please

**Pull request merged**